res/database/Default/database_news.xml

```diff
@@ -227,44 +278,60 @@
 				<pl>Mistrzostwa świata w zapasach WWFr</pl>
 			</title>
 			<description>
-				<de>Die Westamerikanischen Wrestling Freunde (WWFr) veranstalten im Texas Skydome ihren ersten Wrestlewahnsinn. Im Main-Event trifft Badguy „Rowdy“ Pfeiffer auf den Titelträger Hugh Hoden.</de>
-				<en>The West American Wrestling Friends (WWFr) are hosting their first WrestleMadness at the Texas Skydome. In the Main-Event Badguy "Rowdy" Pfeiffer meets the title holder Hugh Hoden.</en>
-				<pl>West American Wrestling Friends (WWFr) organizuje swoje pierwsze pokazy w Texas Skydome. W głównym wydarzeniu Badguy "Rowdy" Pfeiffer spotka się z posiadaczem tytułu Hugh Hodenem.</pl>
+				<de>Die Westamerikanischen Wrestling Freunde (WWFr) veranstalten im Texas Skydome ihren ersten Wrestlewahnsinn. Im Main-Event trifft Bad Guy ${roddy} „${rowdy}“ ${piper} auf den Titelträger ${hulkhogan}.</de>
+				<en>The West American Wrestling Friends (WWFr) are hosting their first WrestleMadness at the Texas Skydome. In the Main-Event, bad guy ${roddy} "${rowdy}" ${piper} meets the title holder ${hulkhogan}.</en>
+				<pl>West American Wrestling Friends (WWFr) organizuje swoje pierwsze pokazy w Texas Skydome. W głównym wydarzeniu ${roddy} „${rowdy}“ ${piper} spotka się z posiadaczem tytułu ${hulkhogan}em.</pl>
 			</description>
-			<data genre="2" price="1" quality="60" fictional="0" flags="2" happen_time="5,1985,1985,3,3,1,15" /> <!-- forerun, actual date 1985-03-31 -->
+			<variables>
+				<roddy>${.person:"9d84be62-ea5a-4142-a9d4-bacf10737c27":"firstname"}</roddy>
```

Comment:
same as in the other instances, for brevity's sake for "known" names.

Reply:
I am with you that famous names will most probably be the same on most countries (similar to movie titles not being translated, or American city names being used in English in Germany but not vice versa).

The issue I see is: if one wants to localize this name, they need to find all occourences. And now it is not just looking up for the guid or the name, it can also be a reference through a cast:index (or role or ..). So it becomes harder to identify what and where to adjust.

Reply:
... I just wanted to point out the consequences of the choice (with emphasis at the very beginning of the refactoring). I do not demand a change.

The cast/role issue may not be too problematic as these references are used in the description (probably with different language versions) and titles (high probability of language versions and if not, it is a very prominent location). For direct references (.self) search for the GUID will locate the licence entry. "${.programme:X:"cast":1:...}" references will be mean, though.

Just like typos, it will be a matter of finding/reporting/fixing...